src/dapp/machine.ts

```diff
@@ -202,6 +214,13 @@ export const blockChainMachine = createMachine<
         ],
       },
     },
+    displayLoading: {
```

Comment:
I would suggest calling this something more related to the functionality. 

`loadingFriendsFarm`